Makefile

```diff
@@ -109,10 +109,10 @@ proxy:
 	go build -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./hack/proxy
 
 run-portal:
-	go run -tags aro -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro portal
+	go run -tags aro,containers_image_openpgp -ldflags "-X github.com/Azure/ARO-RP/pkg/util/version.GitCommit=$(VERSION)" ./cmd/aro portal
 
 build-portal:
-	cd portal && npm install && npm run build
+	cd portal/v1 && npm install && npm run build && cd ../v2 && npm install && npm run build
```

Comment:
I think I see now going further into the review, we are defaulting to v1 with an option to switch to v2.

Reply:
This is a temporary measure for an initial release of the new admin portal just in case there are issues instead of having to revert an RP release. The plan is to eventually remove the v1 admin portal and just keep the old one. Where would be a good place to voice this do you think?